torch/nn/functional.py

```diff
@@ -3910,7 +3910,11 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
             ]
         else:
             assert scale_factors is not None
-            output_size = [int(math.floor(float(input.size(i + 2)) * scale_factors[i])) for i in range(dim)]
+            from ..fx.experimental.symbolic_shapes import sym_int, sym_float
+            output_size = [
+                sym_int(math.floor(sym_float(input.size(i + 2)) * scale_factors[i]))
+                for i in range(dim)
+            ]
```

Comment:
BTW, I kind of think this shouldn't be necessary. And it wouldn't be necessary if dynamo went into these functions, rather than have them as leafs.

Reply:
I agree - I have plans to make this not necessary.

Reply:
Also math.floor() already returns a int. So I think a better fix is to just remove the direct call to int().

Reply:
This is from https://github.com/pytorch/pytorch/commit/666c4f5c87423880fe077f6992994df4e6c34fdc